### PR TITLE
Add IRS indicators Legend to IRS reports

### PR DESCRIPTION
**Pull request by moshthepitt**
Adds a component that explains the colors in IRS reporting tables similar to what was in mSpray - see https://macepa.mspray.onalabs.org/

irs legend2

### CODE REVIEW

File: src/configs/settings.ts [hunk 1]
```diff
@@ -711,35 +712,44 @@ export const symbolLayerConfig = {
...
-    orEquals?: boolean;
-    value: number;
-  };
+  [key: string]: IndicatorThresholdItem;
 }
 
-/** Indicator Thresholds for NA (Namibia) */
-export const indicatorThresholdsNA: IndicatorThresholds = {
+/** IRS Reporting configs */
+export const indicatorThresholdsIRS: IndicatorThresholds = {
```
Comment (cKellyDesign) [inline src/configs/settings.ts]: @moshthepitt should this be `IRS_NA` to support different thresholds and colors for different implementations of the IRS intervention type? (Like if Zambia has different thresholds or if Thailand wants to use different colors, etc?)

Comment (moshthepitt) [reply src/configs/settings.ts]: @cKellyDesign yes it should, although right now ZM and NA are the same values so I thought we should do the configurability when we need it.

Comment (moshthepitt) [reply src/configs/settings.ts]: @cKellyDesign thoughts on that?

Comment (cKellyDesign) [reply src/configs/settings.ts]: Ah yeah ok sounds good, might as well not over engineer it until it's necessary :)

Comment (cKellyDesign) [reply src/configs/settings.ts]: :shipit:

**Pull request merged**